pegasus/sites.v3/code.org/public/helloworld.md.partial

```diff
@@ -37,3 +37,16 @@ In Computer Science 101, the first program many students create is a simple one
 {{ featured_project, title: "Animals", author: "B", age: "18+", image: "images/csc/helloworld/cschelloworld_animals.gif", alt_text: "A student project featuring an animated gif of an underwater scene with dolphin, fish, and jellyfish characters in front of a shipwreck", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN4HAtis4ykKTIjGcNPP9zD4" }}
 {{ featured_project, title: "Retro", author: "W", age: "13+", image: "images/csc/helloworld/cschelloworld_retro.gif", alt_text: "A student project featuring an animated gif of a game of tag with two alien characters on a distant planet", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN7yo2E1S0q87VqlzaBz7oqo" }}
 {{ clearfix }}
+
+<link href="/shared/css/barlow-semi-condensed-font.css" rel="stylesheet"/>
+<link href="/shared/css/phase1-design-system.css" rel="stylesheet"/>
+
+<style>
+h1, h2 {
+    margin-bottom: 16px;
+}
+
+p:first-of-type {
+    margin-bottom: 40px;
+}
+</style>
```

Comment:
These `.md.partial` files are fully-translated by external translators, and shouldn't have any HTML.

Reply:
Thanks for letting me know! This was the only solution I've thought of. Let me check if there's other way.